libvast/src/detail/zeekify.cpp

```diff
@@ -0,0 +1,54 @@
+//    _   _____   __________
+//   | | / / _ | / __/_  __/     Visibility
+//   | |/ / __ |_\ \  / /          Across
+//   |___/_/ |_/___/ /_/       Space and Time
+//
+// SPDX-FileCopyrightText: (c) 2021 The VAST Contributors
+// SPDX-License-Identifier: BSD-3-Clause
+
+#include "vast/detail/zeekify.hpp"
+
+#include "vast/detail/array.hpp"
+#include "vast/logger.hpp"
+
+#include <string>
+
+namespace vast::detail {
+
+using namespace std::string_literals;
+
+record_type zeekify(record_type layout) {
+  // The first field is almost exclusively the event timestamp for standard
+  // Zeek logs. Its has the field name `ts`. For streaming JSON, some other
+  // fields, e.g., `_path`, precede it.
+  for (auto& field : layout.fields)
+    if (field.name == "ts")
+      if (auto ts = caf::get_if<time_type>(&field.type)) {
```

Comment:
Please add braces here if the nested if uses braces, otherwise this becomes really hard to follow.

Reply:
```suggestion
  for (auto& field : layout.fields)
    if (field.name == "ts")
      if (caf::holds_alternative<time_type>(field.type)) {
```